server/server.go

```diff
@@ -652,6 +662,91 @@ func (s *Server) MonitorsCloseChan() chan struct{} {
 	return s.monitorsChan
 }
 
+func (s *Server) startSeccmpNotifierWatcher() error {
```

Comment:
I know it's a long function, but I find it weird that we drop the o in seccomp here

Reply:
was that intentional

Reply:
No, fixed it :+1: